library/testing/TestSDKTriggerInteractions.cxx

```diff
@@ -0,0 +1,129 @@
+#include <engine.h>
+#include <scene.h>
+#include <window.h>
+
+#include "TestSDKHelpers.h"
+
+#include <iostream>
+#include <string_view>
+
+struct TestTriggerHelper
+{
+  TestTriggerHelper(std::string dataPath, std::string baselinePath, std::string outputPath)
+    : DataPath(std::move(dataPath))
+    , BaselinePath(std::move(baselinePath))
+    , OutputPath(std::move(outputPath))
+  {
+  }
+
+  template<typename Func>
+  void operator()(const std::string& name, Func&& func) const
+  {
+    f3d::engine eng = f3d::engine::create();
+    eng.getWindow().setSize(300, 300);
+    eng.getScene().add(this->DataPath);
+    eng.getWindow().render();
+
+    func(eng);
+
+    if (!TestSDKHelpers::RenderTest(eng.getWindow(), this->BaselinePath, this->OutputPath, name))
+    {
+      throw std::runtime_error("Render test failed for " + name);
+    }
+  }
+
+  std::string DataPath;
+  std::string BaselinePath;
+  std::string OutputPath;
+};
+
+int TestSDKTriggerInteractions(int argc, char* argv[])
+{
+  TestTriggerHelper helper(
+    std::string(argv[1]) + "/data/cow.vtp", std::string(argv[1]) + "baselines/", argv[2]);
+
+  try
+  {
+    // Trigger mouse wheel backward and check if it zoomed out
+    helper("TestSDKTriggerInteractionsWheelBackward", [](f3d::engine& engine) {
+      engine.getInteractor().triggerMouseWheel(f3d::interactor::WheelDirection::BACKWARD);
+
+      // following calls are only there for coverage
+      // there is no interactor style action associated with these
+      engine.getInteractor().triggerMouseWheel(f3d::interactor::WheelDirection::LEFT);
+      engine.getInteractor().triggerMouseWheel(f3d::interactor::WheelDirection::RIGHT);
```

Comment:
lets put them both in a dedicated helper call. I dont like that TestSDKTriggerInteractionsWheelBackward is responsible for the coverage of WheelDirection::LEFT